### PR TITLE
EST-21: Footer updates

### PR DESCRIPTION
**Pull request by silviuserdaru**
Updated footer elements / layout, styling and added new links in there.

### CODE REVIEW

**Comment by albertvolkman** [comment]
As noted in Slack, this PR will also need desktop navigation.

**Comment by silviuserdaru** [comment]
Updated header, added two elements in there.
Fixed spacing for EST-37, too (and small other spacing adjustments).

File: _includes/navigation.html [hunk 1]
```diff
@@ -6,24 +6,34 @@
...
+            {% if class %}
+              {% assign class = class | append: " " %}
+            {% endif %}
+
+            {% if nav.url == page.url %}
+              {% assign class = class | append: "is-active" %}
+            {% endif %}
+
+            <li>
+              <a {% if class %} class="{{ class }}"{% endif %} href="{{ nav.url }}">{{ nav.title }}</a>
```
Comment (albertvolkman) [inline _includes/navigation.html:34]: Let's just write this as-
```
<a class="{% if nav.button %}button {% endif %}{% if nav.url == page.url %} is-active{% endif %}" href="{{ nav.url }}">{{ nav.title }}</a>
```

**Comment by albertvolkman** [comment]
Looks like the hamburger menu is showing on desktop-

Screen Shot 2020-06-15 at 12 54 37 PM

**Comment by silviuserdaru** [comment]
Updated. It was showing for like less than half a second, I forced it to not show at all.

File: _includes/navigation.html (hunk 1 above)
Comment (silviuserdaru) [reply _includes/navigation.html]: Updated.

**Comment by albertvolkman** [comment]
Committed here- https://github.com/EsteemedIO/esteemedio.github.io/commit/7ce91455da77b65b0300ea477dd750dc93278cf8

**Pull request closed without merging**